index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Regorus Online Playground</title>
+    <title>Regorus Onine Playground</title>
```

Comment:
Typo. Ok to just use "Regorus Playground"?

Reply:
Fixed.